src/components/RatingsReviews/ReviewTile.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+function ReviewTile ({ review }) {
+
+  function reviewerRec() {
+    if (review.recommend) {
+      return <div> ✔️ I recommend this product </div>
+    }
+  }
+
+  function sellerResponse() {
+    if (review.response) {
+      return (
+        <div className='seller-response'>
+          <p>Seller Response:</p>
+          <p>{review.response}</p>
+        </div>
+      )
+    }
+  }
+
+  function formatDate(dateStr){
+    var options = { year: 'numeric', month: 'long', day: 'numeric' };
+    return new Date(review.date).toLocaleDateString([],options);
+  }
+
+  return (
+    <div className='review-tile'>
+      <div>⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ({review.rating})</div>
+      <div className='reviewer-name-and-date'>
+        <span>{review.reviewer_name}</span>, <span>{formatDate(review.date)}</span>
+      </div>
+
+      <h3 className='review-summary'>{review.summary}</h3>
+      <p className='review-body'>{review.body}</p>
+
+      <div>
+        {review.photos.map((photo, index) =>
+          <img className='review-thumbnail' src={`${photo.url}`} key={photo.id}></img>
+        )}
+      </div>
+
+      <div>{reviewerRec()}</div>
+
+      <div>{sellerResponse()}</div>
+
+      <div className='review-helpfulness-and-report-link'>
```

Comment:
this css name is a little cumbersome i think

Reply:
lol yeah, I think this is better? `vote-helpfulness-and-report`. I think later down the line, the helpfulness vote buttons and report might be separated.